libs/data-service-generator/src/register-plugin.ts

```diff
@@ -36,17 +43,34 @@ async function* getPluginFuncGenerator(
   }
 }
 
+async function getPlugin(
```

Comment:
instead of a function that do that, I would do await import from `${path}packageName`